### PR TITLE
Qmaps 404  poi panel

### PR DESCRIPTION
**Pull request by Julien-laville**

### CODE REVIEW

**Comment by nlehuby** [comment]
The "service & info" text sometimes goes over the arrow:
screenshot_2018-08-02 qwant maps - beta

**Comment by nlehuby** [comment]
It would be easier to understand if we prefix the beer panel with "beers" just like it is done for the internet access info:
strange:
screenshot_2018-08-02 qwant maps - beta 1
much better:
screenshot_2018-08-02 qwant maps - beta 2

**Comment by nlehuby** [comment]
Can we add some tests ?
* click on the POI name in the panel
* click on the service & info block

**Comment by nlehuby** [comment]
screenshot_2018-08-03 qwant maps - beta 1

a `:`after `Bières` is needed here

**Comment by nlehuby** [comment]
The accessibility panel is sometimes empty (due to https://github.com/QwantResearch/idunn/pull/20)
wheelchair

File: src/panel/poi_panel.js [hunk 1]
```diff
@@ -85,6 +85,17 @@ PoiPanel.prototype.setPoi = async function (poi) {
...
 
+PoiPanel.prototype.copy = function () {
+  let copyUrl = document.getElementById('share-url')
+  copyUrl.select()
+  document.execCommand('copy')
+  copyUrl.blur()
+}
+
+PoiPanel.prototype.center = function() {
+  fire('fit_map', this.poi, {sidePanelOffset : true})
```
Comment (amatissart) [inline src/panel/poi_panel.js]: This does not respect the `ZOOM_BY_POI_TYPES` value.  
This is due to how the poi is initialized in `Poi.apiLoad`. The API does not currently return a `type` and the default zoom value is used. 
I think the type `"poi"` should be set by default when loading a Poi from the API.

File: src/views/poi_panel.dot [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="poi_panel {{= this.active ? '' : 'poi_panel--hidden' }}">
   <div class="poi_panel__content">
   {{? this.poi }}
-    <div class="poi_panel__description_container">
+    <div class="poi_panel__description_container" {{= click(this.center, this) }}>
```
Comment (amatissart) [inline src/views/poi_panel.dot]: There is currently no way for the user to notice that a click has an effect here.  
Maybe we could simply change the cursor on hover ?

File: src/panel/poi_panel.js (hunk 1 above)
Comment (Julien-laville) [reply src/panel/poi_panel.js]: fixed

File: src/views/poi_panel.dot (hunk 2 above)
Comment (Julien-laville) [reply src/views/poi_panel.dot]: fixed

**Pull request merged**